index.html

```diff
@@ -73,7 +73,298 @@ <h1>I'm a software developer</h1>
         </ul>
       </div>
       <!-- End of social media -->
+      <div class="button">
+        <a href="#">
+          <img src="images/scroll-button.png" alt="scroll-button" />
+        </a>
+      </div>
+    </section>
+    <!-- *start of work section -->
+    <section class="works">
+      <!-- h1, a line, 6 x cards, 
+        resume, languages, frameworks, skills  -->
+      <div>
+        <h1>My Recent Works</h1>
+      </div>
+      <div class="line"></div>
+      <!-- start of cards -->
+      <!-- <div> -->
+      <!-- h2, 4 x buttons, see-project button -->
+      <div class="card-1">
+        <div class="h2">
+          <h2>
+            Multi-Post Stories<br />
+            Gain+Glory
+          </h2>
+        </div>
+        <div class="card-buttons">
+          <a href="#"
+            ><img class="button1" src="images/ror.png" alt="ror-button"
+          /></a>
+          <a href="#"
+            ><img class="button2" src="images/css.png" alt="css-button"
+          /></a>
+          <a href="#"
+            ><img class="button3" src="images/jvscr.png" alt="javascr-button"
+          /></a>
+          <a href="#"
+            ><img class="button4" src="images/html.png" alt="html-button"
+          /></a>
+        </div>
+        <div class="project-button">
+          <a href="#"><img src="images/greenbtn.png" alt="go-to-project" /></a>
+        </div>
```

Comment:
- [ ] As a project requirement you shouldn't use images (`<img>` tag)for the button and elements, instead, you should use elements to create the structure of your page. 

- [ ] Also, we need to follow the button styling: `Active`, `Hover`, and `Pressed`, Please add these effects for a better user interface.

![Screenshot from 2023-01-14 13-09-34](https://user-images.githubusercontent.com/105117832/212488822-f77a4d77-454f-4c15-a849-e74a1308c3f5.png)